test/spec/cloud-element-templates/properties/CustomProperties.spec copy.js

```diff
@@ -1564,7 +1564,7 @@ describe('provider/cloud-element-templates - CustomProperties', function() {
                 input = domQuery(selector, entry);
 
           // assume
-          expectError(entry, `${name} - MinLength must have min length 5.`);
```

Comment:
This File was accidentally commited in https://github.com/bpmn-io/bpmn-js-element-templates/pull/64 and was removed with 6d2e52476702ae4b5dff8896c313643700069f9a already. Please rebase your work to the latest `main` branch to resolve the conflict